http/src/main/resources/reference.conf

```diff
@@ -939,10 +939,14 @@ terra-app-setup-chart {
   # cert files into the local chart.
   chart-name = "/leonardo/terra-app-setup"
   # If you change this here, be sure to update it in the dockerfile
-  chart-version = "0.0.3"
+  chart-version = "0.0.4"
 }
 
 app-service {
   # Defaults to true, but disabled for fiab runs
   enable-custom-app-check = true
 }
+
+drs {
```

Comment:
The DRS url is now used by Cromwell and Galaxy, so I made it a standalone thing.

Reply:
this will require firecloud-develop change? also cc @andy7i since he'll likely be updating this in firecloud-develop

Reply:
Oh yeah -- I can update fc-develop with the new config structure but keep the values the same.

Reply:
https://github.com/broadinstitute/firecloud-develop/pull/3171 @Qi77Qi